app/controllers/workflows_controller.rb

```diff
@@ -104,51 +106,7 @@ def render_task(task, params)
 
   private
 
-  def ordered_fields(fields)
-    response = Array.new
```

Comment:
This code was used in `set_descriptors_handler` and `set_characterisation_descriptors_handler` only, and in both cases the dependency has been removed.